.pre-commit-config.yaml

```diff
@@ -40,7 +40,6 @@ repos:
   rev: v1.37.1
   hooks:
   - id: yamllint
-    args: [--strict]
```

Comment:
What went wrong with yamllint?

Reply:
everything I'm struggling to get github to run my workflows and for yamllint to be happy about it for some reason... I'll figure it out